### DISCUSSION TITLE
How to log errors to file instead of console?

### DISCUSSION THREAD

**Discussion by betfix** [nextauthjs/next-auth#907] [Help]
Is it possible to avoid logging errors to console and instead log them to file? 

I am using the default TypeOrm Postgres adapter. In the console, there is an error ([next-auth][error][create_user_error] QueryFailedError). Checking console for errors will be a problem when on production. I tried to use the config option "events" -> "error" to call my custom logger but this event is not fired for this particular error.

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [ ] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by balazsorban44** [comment]
When #870 is finished/merged, you should be able to log `next-auth` messages wherever you want!
